### PR TITLE
Fix translation files

### PR DESCRIPTION
**Pull request by syokkendesuyo**
All translation files were checked.

- Add missing key
- Delete unused keys
- Fix wrong escape
etc...

Check diff for details.

### CODE REVIEW

File: src/lang/lwc_cn.properties [hunk 1]
```diff
@@ -22,23 +22,31 @@ you=你
 chest=箱子
 furnace=熔炉
 dispenser=发射器
+dropper=Dropper
 sign=木牌
-wooden_door=橡木门
```
Comment (pop4959) [inline src/lang/lwc_cn.properties]: You shouldn't be deleting valid translations, although for this case you can probably safely rename this to oak_door.

File: src/lang/lwc_cn.properties [hunk 2]
```diff
@@ -22,23 +22,31 @@ you=你
 chest=箱子
 furnace=熔炉
 dispenser=发射器
+dropper=Dropper
```
Comment (pop4959) [inline src/lang/lwc_cn.properties]: Please don't add keys if you are not going to translate them. Untranslated keys default to the English locale anyways. If you do this, the translation will need to be changed in ALL locale files every time an English translation changes. This is very inefficient/tedious.

File: src/lang/lwc_tw.properties [hunk 3]
```diff
@@ -22,9 +18,6 @@ yes=%dark_green%是
 no=%dark_red%否
 
 # Some generic responses and block names
-## TRANSLATORS: 相關翻譯請參閱 https://crowdin.com/project/minecraft/zh_TW
```
Comment (pop4959) [inline src/lang/lwc_tw.properties]: Why are you deleting translator comments? These may be relevant to people who are trying to update the translations.

**Review by pop4959 — CHANGES_REQUESTED** [review]

**Comment by pop4959** [comment]
I didn't review every change, as there were quite a few, and they all for the most part fell under a similar category of problem:

- Adding keys without translating them (defaults will no longer update automatically)
- Deleting keys that were still in use (although updating wooden_door -> oak_door might be appropriate to update it, deleting it and replacing with English is not)
- Removing or changing items that have little or no impact on the translation (for example, comments that are there to help translators)

Changes that may be helpful include:

- Updating outdated keys, and adding or retaining a correct translation
- Cleaning up obsolete keys, such as those that are not used by the plugin and are not likely ever to be used again
- Fixing inconsistent formatting (either in the translation, or in the file) when compared to the default English locale.
- Etc

Overall, you want to consider how your changes are affecting the plugin as a whole, and the people using it. Does your change make the plugin easier to maintain? Does your change positively impact the translation? How does the change affect people who are using the translation?

Lastly, I just want to say that the idea of this PR is really nice. Not many people are willing to spend the time to rearrange ALL of the locale files. Thank you for taking the time to work on this.

**Comment by syokkendesuyo** [comment]
I will correct all of your comments.
But for translators, we should add untranslated phrases.
There is a trade-off between saving translators and developers.

**Comment by pop4959** [comment]
Adding untranslated keys is not saving time for anyone. It only makes it more difficult to maintain. If you want, you could add comments indicating that keys are missing. That would help future translators, but not cause problems with default translations.

**Comment by syokkendesuyo** [comment]
For example, changes have been made to several keys. Is the update of 16 lang files extremely costly and meaningless?

I committed a phrase that was not translated before.
Why was it approved?

Commit:[29cf528](https://github.com/pop4959/LWCX/commit/29cf528e96862b21048f2431464346aa2568b334)

**Comment by pop4959** [comment]
Meaningless? Yes, the plugin automatically handles defaults. Costly? Yes, because obviously updating 16 files takes much more time than just changing one. I can't think of a reason for needing to add untranslated keys. If you think it is important, explain why.

Yes, it looks like the previous PR has the same problem. I was not scrutinising the locale changes as much due to the small, obvious diff. You may wish to revise it in this PR.

Do you have any issue with using comments to indicate missing keys? As I mentioned in my previous reply, this helps future translators while retaining correct default behaviour.

**Comment by syokkendesuyo** [comment]
Although the burden on the developer side is heavy, the translator side should be able to easily notice that it has been changed or added.
I know that with a little effort, the product will be better.

However, it may be a problem I should tolerate because the cultural sphere is different :(

If this topic is not resolved, I on time will be closed because of insufficient.
This is because there are more points to fix.

**Pull request closed without merging**